### DISCUSSION TITLE
Git version install error

### DISCUSSION THREAD

**Discussion by michelefa1988** [GitTools/GitVersion#3596] [Q&A]
Azure pipeline GitVersion install step suddenly started throwing the error bellow.  The install has been running smoothly for a long time all using the same image/os. 

Any ideas?  I am reporting this here before troubleshooting since a few months ago, a similar issue had occurred and it was due to Microsoft updating some default setting,
```
- task: gitversion/setup@0
  displayName: Install GitVersion
  inputs:
    versionSpec: '5.10.3'
```
```

--------------------------
Acquiring GitVersion.Tool version spec: 5.10.3
--------------------------
Command: dotnet tool install GitVersion.Tool --tool-path /agent/_work/_temp --version 5.10.3
/usr/bin/dotnet tool install GitVersion.Tool --tool-path /agent/_work/_temp --version 5.10.3
You must install or update .NET to run this application.

App: /usr/share/dotnet/sdk/7.0.202/dotnet.dll
Architecture: x64
Framework: 'Microsoft.NETCore.App', version '7.0.4' (x64)
.NET location: /usr/share/dotnet/
##[error]Error installing tool

No frameworks were found.

Learn about framework resolution:
https://aka.ms/dotnet/app-launch-failed

To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=7.0.4&arch=x64&rid=ubuntu.22.04-x64
Finishing: Install GitVersion
```

**Comment by HHobeck** [comment]
Dear michelefa,

We doing all our work here in our spare time and are engaged helping people like you. Keeping this in mind it would be very kind to analyse your problems well and not just creating a bug in an open source project without knowing where the bug is located or even knowing if your problem is a bug in git-version or something else.

I'm going to move this issue to the discussion section.

Thank you very much for your understanding.
